src/components/structures/login/Registration.js

```diff
@@ -227,6 +227,9 @@ module.exports = React.createClass({
                     const teamToken = data.team_token;
                     // Store for use /w welcome pages
                     window.localStorage.setItem('mx_team_token', teamToken);
+                    // Set the team token and view homepage
+                    window.matrixChat._teamToken = teamToken;
+                    window.mxDispatcher.dispatch({action: 'view_home_page'});
```

Comment:
We shouldn't be accessing these through the window object. We should get the dispatcher by import `dispatcher` and probably pass the team token up with a callback prop.

Also, I'm a little skeptical about the `view_home_page` dispatch: is there not somewhere else that decides what page to display once registration is complete and can we do this there (are we going to race with it?)